crypto/signature_nocgo.go

```diff
@@ -0,0 +1,156 @@
+// Modifications Copyright 2023 The klaytn Authors
+// Copyright 2017 The go-ethereum Authors
+// This file is part of the go-ethereum library.
+//
+// The go-ethereum library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The go-ethereum library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see <http://www.gnu.org/licenses/>.
+//
+// This file is derived from crypto/signature_cgo.go (2018/06/04).
+// Modified and improved for the klaytn development.
+
+//go:build nacl || js || !cgo || gofuzz
+// +build nacl js !cgo gofuzz
+
+package crypto
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"errors"
+	"fmt"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	btc_ecdsa "github.com/btcsuite/btcd/btcec/v2/ecdsa"
```

Comment:
@hackartists, Please run `go get github.com/btcsuite/btcd/btcec/v2` and push the updated `go.mod` and `go.sum` files.

Reply:
@blukat29 let's merge this and update packages in the other PR